common-lib/src/main/scala/com/gu/mediaservice/lib/auth/Panda.scala

```diff
@@ -20,11 +20,10 @@ trait PanDomainAuthActions extends AuthActions {
   override def authCallbackUrl: String = s"$authCallbackBaseUri/oauthCallback"
 
   override lazy val domain: String = properties("panda.domain")
-  lazy val awsKeyId                = properties("panda.aws.key")
-  lazy val awsSecretAccessKey      = properties("panda.aws.secret")
 
-  override def awsCredentialsProvider = new StaticCredentialsProvider(
-    new BasicAWSCredentials(awsKeyId, awsSecretAccessKey)
+  override def awsCredentialsProvider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("media-service"),
+    new InstanceProfileCredentialsProvider
```

Comment:
It does in theory, but in practise you're unlikely to have both profile credentials and instance profile credentials.